plugins/terraform/data_source_tag_resolver.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/quintilesims/layer0/cli/client"
+)
+
+func resolveTags(client client.Client, target string, params map[string]string) (string, error) {
```

Comment:
Since `type` is a required param for the API, it should probably be a param here instead of relying on the caller to know to use it, e.g.:
```
 resolveTags(client client.Client, target, entityType string, params map[string]string) (string, error) { ... }
```